app/scripts/sagas/index.js

```diff
@@ -128,6 +132,38 @@ export function * fetchPeerIds () {
   }
 }
 
+export function * loadConfig (action) {
+  try {
+    const ipfsConfig = yield call(api.getConfig)
+    const configJSON = JSON.stringify(ipfsConfig, undefined, 2)
```

Comment:
Yes I think it's best to store the draft as unmodified string (as the user entered it) and the the actual config as object.